dspace/config/submission-forms.xml

```diff
@@ -306,6 +323,14 @@
                     <hint>Enter the job title of the person</hint>
                 </field>
             </row>
+            <row>
+                <relation-field>
+                    <relationship-type>isOrgUnitOfPerson</relationship-type>
+                    <search-configuration>organization</search-configuration>
+                    <label>Organization</label>
+                    <hint>Select the Organizational Unit this is part of.</hint>
+                </relation-field>
+            </row>
```

Comment:
Same thing here. Not sure if this would work if `isOrgUnitOfPerson` relationship is not installed on the database.  And, as previously noted, we are not installing these relationships by default (there is no default configuration to install, in fact)

Reply:
I've removed the new config as discussed today, trimming down the PR to fixes only